csfieldguide/templates/interactives/unicode-binary.html

```diff
@@ -0,0 +1,32 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+  <div data-iframe-height>
+    <h4 class="text-center">{% trans 'Unicode to Binary (UTF-<span id="interactive-unicode-binary-mode">8</span>)' %}</h4>
```

Comment:
Previous inputs to this interactive are shown in a dropdown when the box is selected, which then remains stationary relative to the screen when the user scrolls the page
![move](https://user-images.githubusercontent.com/25916475/51946558-c45a9880-2486-11e9-834c-d76e3f10f6a1.png)